test/lib/utils.test.js

```diff
@@ -97,4 +97,12 @@ describe("lib/utils", () => {
 
   });
 
+  describe("feature", () => {
+    it("ldclient-js feature is called", () => {
+      utils.feature("home-test", {}).then( () => {
+        expect(ldClient.feature).toHaveBeenCalled();
```

Comment:
`lsClient.feature` isn't a thing; this should be something like this, right?
```suggestion
        expect(ldClient.variation).toHaveBeenCalled();
```
If the test is passing, then I'm suspicious the promise isn't resolving, so the `expect` never fails.  There is a way to assert the number of assertions within the test (sorry I don't know it off the top of my head).  It might also require making the test body be `async`, as well.

Reply:
That makes sense. I'll verify the test is actually doing what it says it is doing (which like you said it probably isn't if this passes). I'll see what I can find.

Reply:
You were correct in that `identify`, `track` and`feature` functions are not actually passing. [Jest does not wait for promises to resolve](https://jestjs.io/docs/en/tutorial-async), so those tests are essentially skipped. I'm still working out how to fix it. I'm trying to just use async/await but I believe I have to also mock `variation` as well which isn't as easy as I was hoping! Work in progress...